frontend/src/lib/components/TZLabel/TZLabel.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { Meta } from '@storybook/react'
+import { TZLabel } from 'lib/components/TZLabel/index'
+import { now } from 'lib/dayjs'
+import { LemonDivider } from 'lib/lemon-ui/LemonDivider'
+import { useEffect, useState } from 'react'
+
+import { mswDecorator } from '~/mocks/browser'
+
+const meta: Meta<typeof TZLabel> = {
+    title: 'Components/TZ Label',
+    component: TZLabel,
+    parameters: {
+        mockDate: '2023-02-01',
+    },
+    decorators: [mswDecorator({})],
+}
+export default meta
+
+export function Recent(): JSX.Element {
+    return <TZLabel time={now()} />
+}
+
+export function MoreThanADayAgo(): JSX.Element {
+    return <TZLabel time={now().subtract(2, 'day')} />
+}
+
+export function MoreThanADayAgoWithPopover(): JSX.Element {
+    return <TZLabel time={now().subtract(2, 'day')} showPopover={false} />
+}
+
+export function WithMoreThanOne(): JSX.Element {
+    /**
+     * really exists as a regression test...
```

Comment:
Should we write is as an integration test in that case?

Reply:
yeah, lazy of me 🙈

Reply:
(you're right, but I'm going to optimize for progress over "perfection" 🤣 )